content/r/concepts/variables/variables.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'Variables'
+Description: 'A variable is used to store data that can be accessed later by subsequent code.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Variables'
+  - 'Integers'
+  - 'Booleans'
+  - 'Strings'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+A variable is used to store data that can be accessed later by subsequent code. In `R` there are no variable "declaration" commands.
+Variables are instead created with the assignment operator, `<-`. (The more familiar assignment operator, `=`, can be used instead, but is discouraged.)
```

Comment:
```suggestion
A variable is used to store data that can be accessed later by subsequent code. In R, there are no variable "declaration" commands. Instead, they are created with the assignment operator, `<-`. (The more familiar assignment operator, `=`, can be used instead, but is discouraged.)
```

I don't think it is necessary to wrap "R" in back-ticks this isn't the case in other files like `r.md`.